packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -200,7 +200,7 @@ export const makeScheduler = async (
         finishAuctionRound();
         break;
       case 'after':
-        await finishAuctionRound();
+        finishAuctionRound();
```

Comment:
Is this better served by `void` or sinking the promise elsewhere? Is `finishAuctionRound` still returning a promise? Could the error, if it occurs, be communicated to whomever is best positioned to react some other way than awaiting?

Reply:
the use of `await` at all was driven by signaling back a failure of timer `.cancel()`. But that's not necessary: https://github.com/Agoric/agoric-sdk/blob/7842f2dbcb810c25f5cacd99c56ecf03d7386c77/packages/inter-protocol/src/auction/scheduler.js#L168-L170

Without that, there's no reason for any of `clockTick` to be async.

Reply:
That makes sense. Is `finishAuctionRound` still async?

Reply:
no